### PR TITLE
Fix/close decoration

### PR DESCRIPTION
**Pull request by e-reeley**
## Description

This change is used to remove the dashed underline from the 'close' icon beside a meeting or task.

## How To Test This

1) Load the Tasks module
2) Select 'View Tasks' from the side navigation
3) View the 'Close/X' icon beside each row to ensure there is no dashed underline

4) Load the Meetings module
5) Select 'View Meetings' from the side navigation
6) View the 'Close/X' icon beside each row to ensure there is no dashed underline

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by Dillon-Brown** [comment]
Assessed

File: modules/Meetings/Meeting.php [hunk 1]
```diff
@@ -546,9 +546,9 @@ function get_list_view_data() {
 			//cn: added this if() to deal with sequential Closes in Meetings.	this is a hack to a hack(formbase.php->handleRedirect)
 			if(empty($action))
 			     $action = "index";
-            $setCompleteUrl = "<a id='{$this->id}' class='list-view-data-icon' title='".translate('LBL_CLOSEINLINE')."' onclick='SUGAR.util.closeActivityPanel.show(\"{$this->module_dir}\",\"{$this->id}\",\"Held\",\"listview\",\"1\");'>";
-			if ($this->ACLAccess('edit')) {
-                $meeting_fields['SET_COMPLETE'] = $setCompleteUrl . "<span class='suitepicon suitepicon-action-clear'></span></a>";
+            $setCompleteUrl = "<b><a id='{$this->id}' class='list-view-data-icon' title='".translate('LBL_CLOSEINLINE')."' onclick='SUGAR.util.closeActivityPanel.show(\"{$this->module_dir}\",\"{$this->id}\",\"Held\",\"listview\",\"1\");'>";
```
Comment (samus-aran) [inline modules/Meetings/Meeting.php]: @e-reeley Why b tag though? Is this consistent with other a href in list views? I see we have this list-view in Calls aswell, are not looking at that also?

Comment (e-reeley) [reply modules/Meetings/Meeting.php]: @samus-aran Here ive used the b tag to override the style which seems to be applied to most columns within table which underlines the populated values. This applies a different style to the field. I could search another fix if this way is not most efficient? 

Ill also have a look and alter the other list views that I have missed and add them into this pr. Thanks!

**Comment by Dillon-Brown** [comment]
Assessed

**Pull request merged**